src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,597 @@
+
+| Parameter            | Type             | Description                                                                                      |
+| ---------------------| ---------------- | ------------------------------------------------------------------------------------------------ |
+| amount               | string           | Amount of tokens transferred.                                                                    |
+| block_hash           | string           | Hash of block in which transfer occurred.                                                        |
+| block_number         | integer          | Height of block in which transfer occurred.                                                      |
+| block_timestamp      | string           | Block date/time string.                                                                          |
+| chain                | string           | Chain name. One of `AVALANCHE`, `BSC`, `ETH`, `FANTOM`, or `POLYGON`.                            |
+| collection_name      | string           | May be `null`. Name of collection which token belongs to.                                        |
+| contract_type        | string           | Contract type. `ERC721` or `ERC1155`.                                                            |
+| token_address        | string           | Address of token transferred.                                                                    |
+| token_id             | string           | Token ID.                                                                                        |
+| possible_spam        | boolean          | May be `null`. If `true`, the contract address is has [been identified](https://docs.moralis.io/web3-data-api/evm/nft-spam-detection) as associated with spam, phishing attempts, or suspicious activities. |
+| transaction_hash     | string           | Transaction ID of the transfer.                                                                  |
+| transaction_index    | string           | Transaction index of the transfer.                                                               |
+| log_index            | string           | Log index of the transfer.                                                                       |
+| value                | string           | Tranfer value.                                                                                   |
+| transaction_type     | string           | Transation type. Possible values are `Single`.                                          |
+| from_address         | string           | Address of previous owner which sent the token(s).                                               |
+| to_address           | string           | Address of new owner which received the token(s).                                                |
+| status               | string           | Transfer status. Will be either `Send` or `Receive`.                                             |
```

Comment:
Please add a note, that if there is the situation that if user sent NFT to them self (from to addresses are  identical), then status also will be `Receive`
it is my comment from the code.
`if my_wallet == from_address && my_wallet == to_address it is incoming tx`

Reply:
thanks, added.